api/PannerNode.json

```diff
@@ -0,0 +1,985 @@
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "setPosition": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PannerNode/setPosition",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "14"
+            },
+            "chrome_android": {
+              "version_added": "14"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
+            },
+            "firefox_android": {
+              "version_added": "26"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "setVelocity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PannerNode/setVelocity",
+          "support": {
+            "webview_android": {
+              "version_added": "14",
+              "version_removed": "51"
+            },
+            "chrome": {
+              "version_added": "14",
+              "version_removed": "51"
+            },
+            "chrome_android": {
+              "version_added": "14",
+              "version_removed": "51"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "25",
+              "version_removed": true
+            },
+            "firefox_android": {
+              "version_added": "26",
+              "version_removed": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15",
+              "version_removed": "38"
+            },
+            "opera_android": {
+              "version_added": "15",
+              "version_removed": "38"
+            },
+            "safari": {
+              "version_added": "6",
+              "version_removed": true
```

Comment:
Not removed https://github.com/WebKit/webkit/blob/master/Source/WebCore/Modules/webaudio/PannerNode.idl#L53